src/components/PoolOverview.tsx

```diff
@@ -75,141 +86,197 @@ export default function PoolOverview({
+          <Typography variant="subtitle1">TVL</Typography>
+          <Typography component="span">{`$${formattedData.reserve}`}</Typography>
 
-      <div className="right">
-        <div className="poolInfo">
+          {formattedData.volume && (
+            <div>
+              <Typography variant="subtitle1">{`${t(
+                "24HrVolume",
+              )}`}</Typography>
+              <Typography component="span">{formattedData.volume}</Typography>
+            </div>
+          )}
+        </Grid>
+        <Grid item xs={6} lg={2.5}>
           {poolData.sdlPerDay != null && IS_SDL_LIVE && (
-            <div className="margin">
-              <span className="label">
-                <a
+            <Box sx={{ display: "flex", alignItems: "center" }}>
+              <Typography variant="subtitle1" mr={1}>
+                <Link
                   href="https://blog.saddle.finance/introducing-sdl"
                   target="_blank"
                   rel="noreferrer"
-                  style={{ textDecoration: "underline" }}
+                  sx={{ textDecoration: "underline" }}
                 >
                   SDL/24h
-                </a>
-              </span>
-              <span>
-                <img src={logo} className="tokenIcon" />
-                {formattedData.sdlPerDay}
-              </span>
-            </div>
+                </Link>
+              </Typography>
+              <img src={logo} className="tokenIcon" width="24px" />
+              :&nbsp;
+              <Typography>{formattedData.sdlPerDay}</Typography>
+            </Box>
           )}
           {formattedData.apy && (
             <div className="margin">
-              <span className="label">{`${t("apy")}`}</span>
-              <span>{formattedData.apy}</span>
+              <Typography component="span" variant="subtitle1">
+                {`${t("apy")}`}:{" "}
+              </Typography>
+              <Typography component="span">{formattedData.apy}</Typography>
             </div>
           )}
           {Object.keys(poolData.aprs).map((key) => {
             const symbol = poolData.aprs[key as Partners]?.symbol as string
             return poolData.aprs[key as Partners]?.apr.gt(Zero) ? (
               <div className="margin Apr" key={symbol}>
                 {symbol.includes("/") ? (
-                  <span className="label underline">
+                  <Typography
+                    component="span"
+                    variant="subtitle1"
+                    sx={{ borderBottom: "1px dotted" }}
+                  >
                     <Tooltip
                       title={
                         <React.Fragment>
                           {symbol.replaceAll("/", "\n")}
                         </React.Fragment>
                       }
                     >
-                      <React.Fragment>Reward APR</React.Fragment>
+                      <React.Fragment>Reward APR: </React.Fragment>
```

Comment:
Could you remove the Fragment and wrap the tooltip around the `Typography` here?

Reply:
That way is more obvious even though I have already gave font for tooltip on `theme.ts`!

Reply:
Fixed!